app/helpers/check_answer_url_helper.rb

```diff
@@ -6,16 +6,6 @@ def check_answer_url_for(journey_type, field_name, application = nil, args = {})
       legal_aid_application: application,
       params: args,
     )
-    anchor = field_name_to_anchor_map[field_name]
```

Comment:
This is weird... it implies this was once a single page where the CYA could link to a specific field 🤔 
Are we reverting to what was, and should we be linking to a specific field again?